contracts/DebtToken.sol

```diff
@@ -106,7 +117,7 @@ contract DebtToken is Ownable {
    * @dev Function to stop minting new tokens.
    * @return True if the operation was successful.
    */
-  function finishMinting() onlyOwner private returns (bool) {
+  function finishMinting() onlyBorrower private returns (bool) {
```

Comment:
`private` should be `internal`.
We do not want it called except by this contract.
Private allows any contract that uses this contract call it.

Reply:
`private` means that only current contract can call it, `internal` allows the contract that inherits from the contact.

We indeed can change it to `internal` shall be better option for the future and building on top of debt contract.